src/vscode-bicep/src/commands/decompile.ts

```diff
@@ -67,9 +67,8 @@ export class DecompileCommand implements Command {
       );
     }
 
-    const canDecompile = await DecompileCommand.mightBeArmTemplateNoThrow(
-      documentUri,
-    );
+    const canDecompile =
```

Comment:
Changes to `src/vscode-bicep` are non-functional - I just ran `npm run lint:fix` to clear the build warnings that are showing up on every PR.